src/ir/unmarshal.cpp

```
@@ -729,4 +729,22 @@ std::variant<InstructionSeq, std::string> unmarshal(const RawProgram& raw_prog,
     return unmarshal(raw_prog, notes, options);
 }
 
+int size(const Instruction& inst) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)